source/exe/main_common.h

```diff
@@ -65,7 +66,10 @@ class MainCommonBase {
                     const AdminRequestFn& handler);
 
 protected:
-  ProcessWide process_wide_; // Process-wide state setup/teardown.
+  ProcessWide process_wide_; // Process-wide state setup/teardown (excluding grpc).
+#ifdef ENVOY_GOOGLE_GRPC
```

Comment:
if we do ifdef guard here, do we still need ifdef guard in GoogleGrpcContext?

Reply:
Actually I removed it here and kept it in GoogleGrpcContext.